src/profilo.php

```diff
@@ -144,7 +144,7 @@ function isSaldoOverflow($database, $utente, $saldo) {
                     $avvisoSaldo .= $database->executeCRUDPreparedStatement($query, 'ds', $values);
                 }
                 else {
-                    $avvisoSaldo .= makeMessageParagraph("Questa aggiunta potrebbe sforare il tetto massimo del saldo possedibile, che sarebbe 99999.99999!");
+                    $avvisoSaldo .= makeMessageParagraph("Questa aggiunta potrebbe sforare il tetto massimo del saldo posseduto, che sarebbe 99999.99999!");
```

Comment:
```suggestion
                    $avvisoSaldo .= makeMessageParagraph("Questa aggiunta potrebbe sforare il tetto massimo del saldo possedibile, che sarebbe 99999.99999!");
```
In italiano, se non erro, è giusto "possedibile"